### PR TITLE
zip.js with npm

### PR DESCRIPTION
**Pull request by ebogo1**
Moves zip.js to use the npm module.

Opening as a draft to make sure I'm on the right track here. Will leave a review with some comments and questions.

@mramato @lilleyse

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ebogo1!
* :heavy_check_mark: Signed CLA found.
* :grey_question: Changes to third party files were made.
    * Looks like a file in one of our ThirdParty folders (`ThirdParty/, Source/ThirdParty/`) has been added or modified. Please verify that it has a section in [LICENSE.md](https://github.com/CesiumGS/cesium/blob/main/LICENSE.md) and that its license information is up to date with this new version.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: ThirdParty/npm/zip.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import * as zip from "@zip.js/zip.js";
+// zip.configure({
```
Comment (ebogo1) [inline ThirdParty/npm/zip.js:2]: I couldn't figure out how to link to the `pako` files here, but it should be possible to use zip.js with the `pako` files that we already install with npm per the [`@zip.js/zip.js` docs](https://gildas-lormeau.github.io/zip.js/core-api.html#configuration).

File: Source/DataSources/exportKml.js [hunk 2]
```diff
@@ -20,14 +20,31 @@ import HeightReference from "../Scene/HeightReference.js";
...
 import CompositePositionProperty from "./CompositePositionProperty.js";
 import ModelGraphics from "./ModelGraphics.js";
 import RectangleGraphics from "./RectangleGraphics.js";
 import SampledPositionProperty from "./SampledPositionProperty.js";
 import SampledProperty from "./SampledProperty.js";
 import ScaledPositionProperty from "./ScaledPositionProperty.js";
+import zip from "../ThirdPartyNpm/zip.js";
+import pako from "../ThirdPartyNpm/pako.js";
+
+var DeflateInflate = zip.initShimAsyncCodec(
```
Comment (ebogo1) [inline Source/DataSources/exportKml.js:33]: This is a placeholder call to an init function that does not use workers but I imagine it's better to use the `pako` ones we already have (see my `pako` comment). 

Should the call to the zip configuration function live in the ThirdParty/npm file?

File: Source/DataSources/exportKml.js [hunk 3]
```diff
@@ -323,54 +340,34 @@ function exportKml(options) {
 }
 
 function createKmz(kmlString, externalFiles) {
```
Comment (ebogo1) [inline Source/DataSources/exportKml.js:342]: If these changes look good, I'll make similar ones in `KmlDataSource` and this should be good to merged once the worker config is figured out.

File: ThirdParty/npm/zip.js (hunk 1 above)
Comment (lilleyse) [reply ThirdParty/npm/zip.js:2]: First thing I would check is the paths

> The first URL is relative to the base URI of the document. The other URLs are relative to the URL of the first script. 

But do we actually need zip.js to use the pako files? zip.js runs its own workers by default. I assume that this is what was already happening but I could be wrong or misremembering from when we talked with @mramato.

File: Source/DataSources/exportKml.js (hunk 3 above)
Comment (lilleyse) [reply Source/DataSources/exportKml.js:342]: Cool, I had a few comments. Let me know when `KmlDataSource` is ready and I'll take a look.

File: ThirdParty/npm/zip.js (hunk 1 above)
Comment (ebogo1) [reply ThirdParty/npm/zip.js:2]: I wonder if the original files we have in Workers/ in `main` are from an old version of zip-js? Looking at the source of https://github.com/gildas-lormeau/zip.js/releases/tag/v2.0.0 it looks like that's the case.

Comment (lilleyse) [reply ThirdParty/npm/zip.js:2]: Can `deflate.js` and `inflate.js` be removed then? Also can the commented code here be removed?

Comment (ebogo1) [reply ThirdParty/npm/zip.js]: From slack, I think it's fine to remove them. The default behavior of the zip.js package is to use web workers, and in Sandcastle/tests it seems to work fine without any additional config options. I removed them in 7b40a60.

**Pull request merged**